conftest.py

```diff
@@ -35,7 +36,18 @@ def create_database_connections():
 
 @pytest.fixture
 def sample_dag():
-    yield DAG("test_dag", default_args={"owner": "airflow", "start_date": DEFAULT_DATE})
+    try:
+        repo = git.Repo(search_parent_directories=True)
```

Comment:
@dimberman  - In case of the PR from A to C, the `head` will be pointing to head of A branch or C branch?

Example - 
A - |
     C
B - |